src/main/webapp/app/exercises/programming/hestia/testwise-coverage-report/testwise-coverage-file.component.scss

```diff
@@ -0,0 +1,17 @@
+.mat-expansion-panel-header {
+    padding: 0;
+}
+
+::ng-deep .mat-expansion-panel-body {
```

Comment:
ng-deep is apparently deprecated. You may instead prevent style encapsulation (ViewEncapsulation.None) on the module that this CSS class is used and annotate properties of this class with `!important`. Below you may see a similar implementation of mine;

CSS File; https://github.com/ls1intum/Artemis/blob/dbbb0c0a77a76a9150a1d104e4596bc9bd2c14cf/src/main/webapp/app/shared/metis/posting-inline-input/message-inline-input.component.scss
Component where encapsulation of styles are disabled; https://github.com/ls1intum/Artemis/blob/dbbb0c0a77a76a9150a1d104e4596bc9bd2c14cf/src/main/webapp/app/shared/metis/posting-inline-input/message-inline-input.component.ts

Reply:
Thanks for the hint, using `.mat-expansion-panel` actually also did the job for me 👍